src/bp/core/security/router-security.ts

```diff
@@ -218,3 +219,37 @@ const checkPermissions = (workspaceService: WorkspaceService) => (
 
   audit(debugSuccess, { userRole: role?.id })
 }
+
+const mediaPathRegex = new RegExp(/^\/api\/v(\d)\/bots\/[A-Z0-9_-]+\/media\//, 'i')
```

Comment:
The reason why it's declared there is that the RegExp is parsed only once, while if we move it inside the function, it would be parsed each and every time the method is called